css/index.css

```diff
@@ -137,10 +279,165 @@ footer nav {
     align-items: center;
     padding: 20px 2%;
-    font-size: 14px;
+    font-size: 1.4 rem;
 }
 
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+/* Tablet*/
+@media (max-width: 768px) {
+    .container {
+        width: 100%;
+    }
+
+header  {
+    width: 100%;
+    display: flex;
+    margin: 20px 0 15px;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-content: center;
+}
+
+header nav {
+    width: 100%;
+    display: flex;
+    justify-content: space-around;
+    margin-top: 30px;
+}
+
+.middle-content .boxes .box1 {
+    width: 12.5%;
+    min-width: 100px;
+    height: 100px;
+    background: teal;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.box2 {
+    width: 12.5%;
+    min-width: 100px;
+    height: 100px;
+    background: forestgreen;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center; 
+}
+
+.box3 {
+    width: 12.5%;
+    min-width: 100px;
+    height: 100px;
+    background: gold;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center; 
+}
+
+.box4 {
+    width: 12.5%;
+    min-width: 100px;
+    height: 100px;
+    background: darkorchid;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center; 
+}
+
+.box5 {
+    width: 12.5%;
+    min-width: 100px;
+    height: 100px;
+    background: cadetblue;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center; 
+}
+
+.box6 {
+    width: 12.5%;
+    min-width: 100px;
+    height: 100px;
+    background: hotpink;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center; 
+   
+}
+
+.box7 {
+    width: 12.5%;
+    min-width: 100px;
+    height: 100px;
+    background: coral;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center; 
+}
+
+.box8 {
+    width: 12.5%;
+    min-width: 100px;
+    height: 100px;
+    background: indigo;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center; 
+}
+
+.box9 {
+    width: 12.5%;
+    min-width: 100px;
+    height: 100px;
+    background: crimson;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center; 
+}
+
+.box10 {
+    width: 12.5%;
+    min-width: 100px;
+    height: 100px;
+    background: lawngreen;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center; 
+}
+
+}
+
+/* Phone */
+@media (max-width: 400px) {
+   .middle-content .boxes {
+       flex-direction: row-reverse;
+     }
+     .middle-content .boxes .box10 {
+         display: none;
+     } 
```

Comment:
I would have loved to see the menu go vertical like the design files at the 400px breakpoint.
![horizontal](https://user-images.githubusercontent.com/12449668/37313937-a8257986-260e-11e8-9112-6302491db22b.png)

Vs.

![vertical](https://user-images.githubusercontent.com/12449668/37313968-c221aab2-260e-11e8-893c-37cc2d000c8e.png)